### ISSUE TITLE
cmek_settings.yaml Issue

### ISSUE THREAD

**Issue by Red-Five** [forseti-security/policy-library#188]
I copied all samples to my policy/constraints folder.

Config validator Pod wont start due to error
Failed to load server Conflicting constraint metadata names with name cmek_rotation from file /policy-library/policies/constraints/cmek_settings.yaml

I see two constraints using same name: cmek_rotation
grep cmek_rotation *.yaml                                                           
cmek_rotation.yaml:  name: cmek_rotation
cmek_settings.yaml:  name: cmek_rotation

Suggest that examples be written so that new users can just copy them all and run without issue.

**Comment by morgante** [comment]
Agreed, we should be sure that samples can be directly copied/applied without needing updates as much as possible.
